.bin/cleanup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set +ex
+SITE_ENV="${TERMINUS_SITE}.ci-${BUILD_NUM}"
+# Nuke any lingering multidev environments from orbit.
+for ENV in $(terminus multidev:list --field=Name --format=list $TERMINUS_SITE); do
+  if [[ $ENV == ci-* ]]; then
```

Comment:
🔢 shellcheck would complain about unquoted variables. This comment is not specifically about the quoting, but that it's up to you if you want to apply shellcheck to .bin. I'm generally in favor it.